docs/lib/datastore/datastore-events.md

```diff
@@ -52,7 +52,8 @@ HubPayload: N/A
 Dispatched when a local change has been newly staged for synchronization with the Cloud
 
 HubPayload `outboxMutationEvent` contains:
-- `modelName` (String): the name of the model that is awaiting publication to the Cloud
+- `model`:
+    - `name` (String): the name of the model that was synced
```

Comment:
Did you mean to replace the descriptions here and for the one below?

Reply:
Bad copypasta 🍝.. Good catch!

Reply:
Fixed!